src/utils/formHelperFunctions.js

```diff
@@ -52,14 +48,14 @@ const formatStepFields = (step, fieldValues, signatureRef) => {
 
     const servar = field.servar;
     let value;
-    if (servar.type === 'signature') {
+    // Only use base64 for signature if these values will be presented to the user
+    if (servar.type === 'signature' && forUser) {
```

Comment:
super nit: forUser maybe should go first so type doesn't even need to get evaluated for code paths that aren't for the user